tests/functional/c/consider/consider_using_augmented_assign.py

```diff
@@ -0,0 +1,62 @@
+"""Tests for consider-using-augmented-assign."""
+
+# pylint: disable=invalid-name,too-few-public-methods,import-error,consider-using-f-string
+
+from unknown import Unknown
+
+x = 1
+x = x + 1  # [consider-using-augmented-assign]
+x = 1 + x  # [consider-using-augmented-assign]
+x, y = 1 + x, 2 + x
+# We don't warn on intricate expressions as we lack knowledge
+# of simplifying such expressions which is necessary to see
+# if they can become augmented
+x = 1 + x - 2
+x = 1 + x + 2
+
+# For anything other than a float or an int we only want to warn on
```

Comment:
Primer pointed this out. Appending list and changing strings is all order dependent. I think it makes sense to be a little stricter here to avoid false positives.

Reply:
If it's easy enough, could you share the primer link / output that helped you see this? I'm trying to understand how you maintainers are using primer as feedback

Reply:
It's really just manually going through the results. The previous comment by the primer (https://github.com/PyCQA/pylint/pull/7514#issuecomment-1255174121) has now been hidden but case 6 of `sentry` triggered me to think about the implications of list concatenation and how those are order dependent (in contrast to simple integer calculation).

It's a bit hands on and for larger new checks like this it is a bit much, but sometimes the `primer` shows only one or two changes for a PR for which we do no expect any changes. That can be a sign that the PR would introduce a new false positive or is actually a previously unknown false negative.